ddtrace/contrib/flask/patch.py

```diff
@@ -128,6 +129,13 @@ def _request_span_modifier(self, span, environ):
         req_body = None
         if config._appsec_enabled and request.method in _BODY_METHODS:
             content_type = request.content_type
+            wsgi_input = environ.get("wsgi.input", "")
+
+            # Copy wsgi input if not seekable
+            if not hasattr(wsgi_input, "seekable") or not wsgi_input.seekable():
```

Comment:
```suggestion
            try:
                seekable = wsgi_input.seekable()
            except AttributeError:
                seekable = False
            if not seekable:
```
so that we can save a second attribute lookup later on?